docker/db/scripts/2023-07-12-reset-users.sql

```diff
@@ -0,0 +1,2 @@
+update "List"
```

Comment:
I couldn't tell you how exactly without looking inside the docker image (i.e. what does the sh script look like), but gist of it is for the postgres docker images, anything found in /docker-entrypoint-initdb.d/* will be executed. 

https://hub.docker.com/_/postgres

>If you would like to do additional initialization in an image derived from this one, add one or more *.sql, *.sql.gz, or *.sh scripts under /docker-entrypoint-initdb.d (creating the directory if necessary). After the entrypoint calls initdb to create the default postgres user and database, it will run any *.sql files, run any executable *.sh scripts, and source any non-executable *.sh scripts found in that directory to do further initialization before starting the service.